lib/matplotlib/tests/test_cycles.py

```diff
@@ -104,6 +126,45 @@ def test_fillcycle_ignore():
     ax.legend(loc='upper left')
 
 
+@cleanup
+def test_valid_input_forms():
+    fig, ax = plt.subplots()
+    # These should not raise an error.
+    ax.set_prop_cycle(None)
+    ax.set_prop_cycle(cycler('linewidth', [1, 2]))
+    ax.set_prop_cycle('color', 'rgywkbcm')
+    ax.set_prop_cycle('linewidth', (1, 2))
+    ax.set_prop_cycle('linewidth', [1, 2])
+    ax.set_prop_cycle('linewidth', iter([1, 2]))
+    ax.set_prop_cycle('linewidth', np.array([1, 2]))
+    ax.set_prop_cycle('color', np.array([[1, 0, 0],
+                                         [0, 1, 0],
+                                         [0, 0, 1]]))
+    ax.set_prop_cycle(lw=[1, 2], color=['k', 'w'], ls=['-', '--'])
+    ax.set_prop_cycle(lw=np.array([1, 2]),
+                      color=np.array(['k', 'w']),
+                      ls=np.array(['-', '--']))
+    assert True
+
+
+@cleanup
+def test_invalid_input_forms():
+    fig, ax = plt.subplots()
+    with assert_raises((TypeError, ValueError)):
```

Comment:
Right, sorry. That was a last minute change that went into 1.5 prior to
release. Carry on.

On Mon, Nov 9, 2015 at 4:44 PM, u55 <notifications@github.com> wrote:

> In lib/matplotlib/tests/test_cycles.py
> <https://github.com/matplotlib/matplotlib/pull/5452#discussion_r44336801>:
>
> > +    ax.set_prop_cycle('linewidth', iter([1, 2]))
> > +    ax.set_prop_cycle('linewidth', np.array([1, 2]))
> > +    ax.set_prop_cycle('color', np.array([[1, 0, 0],
> > +                                         [0, 1, 0],
> > +                                         [0, 0, 1]]))
> > +    ax.set_prop_cycle(lw=[1, 2], color=['k', 'w'], ls=['-', '--'])
> > +    ax.set_prop_cycle(lw=np.array([1, 2]),
> > +                      color=np.array(['k', 'w']),
> > +                      ls=np.array(['-', '--']))
> > +    assert True
> > +
> > +
> > +@cleanup
> > +def test_invalid_input_forms():
> > +    fig, ax = plt.subplots()
> > +    with assert_raises((TypeError, ValueError)):
>
> According to the what's new page
> <https://github.com/matplotlib/matplotlib/blob/838502e7ccf5e97e30e3f80db15e00eb124fe600/doc/users/whats_new.rst>,
> matplotlib 1.5 dropped support for python 2.6.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/5452/files#r44336801>.
>